include/swift/AST/ProtocolConformanceRef.h

```diff
@@ -158,6 +178,9 @@ class ProtocolConformanceRef {
     return ProtocolConformanceRef(UnionType::getFromOpaqueValue(value));
   }
 
+  /// Retrieve the conforming type.
+  Type getConformingType() const;
+
   /// Return the protocol requirement.
   ProtocolDecl *getRequirement() const;
```

Comment:
And if you rename this to getProtocol(), again for consistency with ProtocolConformance, I will forever be grateful